03_launch_mgmt_cluster.sh

```diff
@@ -173,6 +173,11 @@ EOF
     echo "IRONIC_KERNEL_PARAMS=console=ttyS0" | sudo tee -a "${IRONIC_DATA_DIR}/ironic_bmo_configmap.env"
   fi
 
+  # TODO (mboukhalfa) enable heartbeating and ironic TLS when sushy-tools release v1.3.1
+  if [[ "${NODES_PLATFORM}" == "fake" ]]; then
+    echo "OS_AGENT__REQUIRE_TLS=false" | sudo tee -a "${IRONIC_DATA_DIR}/ironic_bmo_configmap.env"
```

Comment:
In a follow up PR, can you make `${IRONIC_DATA_DIR}/ironic_bmo_config.env}` a variable and use it instead of hardcoding the filename. It is repeated good number of times now and is prone to errors.

Same with the target location `${BMOPATH}/ironic-deployment/components/keepalived/ironic_bmo_configmap.env`